cookbook_se/recipes/05_native_plugins/pytorch_mnist.py

```diff
@@ -243,4 +249,30 @@ def pytorch_training_wf(hp: Hyperparameters) -> (PytorchPickledFile, PNGImage):
 # It is possible to run the model locally with almost no modifications (as long as the code takes care of the resolving
 # if distributed or not)
 if __name__ == "__main__":
-    print(pytorch_training_wf(hp=Hyperparameters(epochs=2, batch_size=128)))
+    model, plot, logs = pytorch_training_wf(hp=Hyperparameters(epochs=2, batch_size=128))
+    print(f"Model: {model}, plot PNG: {plot}, Tensorboard Log Dir: {logs}")
+
+# %%
+#
+# .. _pytorch_tensorboard:
+#
+# Rendering the output logs in tensorboard
+# -----------------------------------------
+# When running locally, the output of execution looks like
+#
+# .. code-block::
+#
+#   Model: /tmp/flyte/20210110_214129/mock_remote/8421ae4d041f76488e245edf3f4360d5/my_model.h5, plot PNG: /tmp/flyte/20210110_214129/mock_remote/cf6a2cd9d3ded89ed814278a8fb3678c/accuracy.png, Tensorboard Log Dir: /tmp/flyte/20210110_214129/mock_remote/a4b04e58e21f26f08f81df24094d6446/
+#
+# You can use the ``Tensorboard Log Dir: /tmp/flyte/20210110_214129/mock_remote/a4b04e58e21f26f08f81df24094d6446/`` as
+# an input to tensorboard to visualize the training as follows
+#
+# .. prompt:: bash
+#
+#   tensorboard --logdir /tmp/flyte/20210110_214129/mock_remote/a4b04e58e21f26f08f81df24094d6446/
+#
+#
+# If running remotely (executing on Flyte hosted environment), the workflow execution outputs can be retrieved.
+# Refer to .. TODO.
```

Comment:
This is flytectl right?  Which kinda begs the question of what to do about the rest of the ux.  downloading inputs and outputs of a task or wf run is not all that dissimilar from launching a new run of something, which is not that different than registering i feel like.  and once you get to that you venture into the realm of interactive/notebook based ux, which i think is something we need to address somewhat soonish.